actionlist/app/containers/ActionListContainer.js

```diff
@@ -1,17 +1,69 @@
-import React from 'react';
-
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
 import {
   View,
+  DrawerLayoutAndroid,
 } from 'react-native';
 
 import Header from '../components/header';
 import DisplayList from '../components/display_list';
+import SideMenu from '../components/SideMenu';
+import { toggleDrawer } from '../actions/action_items';
+
+class ActionListContainer extends Component {
```

Comment:
Oh, haha, sorry, I didn't realize you were the one who posted that question. I'm still confused about what you're doing here. You only seem to be using `DrawerLayoutAndroid.onDrawerClose` to trigger a toggle event, which doesn't make sense to me. Shouldn't there be a `DRAWER_OPENED` and a `DRAWER_CLOSED` action to mimic the behavior of the native component? Also, why does this container component care about the state of the native component since the application isn't exhibiting any special behavior based on whether the drawer is open or closed? Doesn't the native component handle whether it's rendered open or closed, why do you need to manually call `openDrawer()` or `closeDrawer()` in `componentDidUpdate`?